os/arch/arm/src/imxrt/imxrt_pwm.c

```diff
@@ -78,6 +78,15 @@
 #include "chip/imxrt_memorymap.h"
 #include "imxrt_pwm.h"
 
+#if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+#define IMXRT_PWM_CNT 2
+#elif defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT105x)
+#define IMXRT_PWM_CNT 4
```

Comment:
Why don't you use `PWM_CNT_COUNT`? Why do you define another definition for the same?